shap/explainers/_explainer.py

```diff
@@ -1,4 +1,5 @@
 import copy
+from sqlite3 import NotSupportedError
```

Comment:
is this needed?

Reply:
Accidental import! pesky auto importers :) good catch